src/screens/DiscoveryScreenV2/elements/DiscoveryItemList.js

```diff
@@ -108,7 +109,7 @@ const DomainList = (props) => {
               {item.name}
             </Text>
 
-            {item.description !== null && (
+            {((!!item.user_id_follower && !!item.description) || isDomain) && (
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This line has a complex condition that might be hard to understand at first glance. Consider breaking it down into well-named variables for better readability.
```diff
- {((!!item.user_id_follower && !!item.description) || isDomain) && (
+ const hasFollowerAndDescription = !!item.user_id_follower && !!item.description;
+ const shouldDisplayText = hasFollowerAndDescription || isDomain;
+ {shouldDisplayText && (
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->